### ISSUE TITLE
Support: OIDC connecter throws errors after working good for a while

### ISSUE THREAD

**Issue by Katz-Tal** [MISP/MISP#9174]
### Support Questions

I'm using the OIDC connecter for a while to integrate MISP login with Okta.
It worked well for months now.
But suddenly it stopped working and I'm seeing the following error:
```
JakubOnderka\ErrorResponse] Error 'invalid_request' received from IdP: Cannot supply multiple client credentials
```

I don't have multiple credentials on the config file, only the same one I used before.
I tried everything and nothing worked.
Also, there isn't enough information about people using it :( 

### MISP version

2.4.170

### Operating System

Ubuntu

### Operating System version

22.04

### PHP version

7.4

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by chennin7** [comment]
Yeah we're seeing the same thing when we tried to set this up initially this week. Tried a ton of stuff and we still can't get it to work.

**Comment by Katz-Tal** [comment]
Can you please update me if somehow you manage to solve it? @chennin7

**Comment by egypcio** [comment]
that sadly never really worked well for me, and other folks I am in contact with.

shared a bit more information on a related issue:
- https://github.com/MISP/MISP/issues/8598#issuecomment-2587332066

**Comment by srinathman** [comment]
it worked for me with Okta after setting these:

``` 
    'issuer' => 'https://<IDP_URI>',
     'redirect_uri' => 'https://<MISP_URL>/users/login',
     'authentication_method' => 'client_secret_post',
```
